### PR TITLE
Update RemoteEnvironment and EventActivation queries after breaking changes

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Update RemoteEnvironment and EventActivation queries after breaking changes

**Related issue(s)**
 - https://github.com/kyma-project/kyma/issues/723

### CODE REVIEW

File: components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go [hunk 1]
```diff
@@ -30,26 +30,21 @@ func (c *remoteEnvironmentConverter) ToGQL(in *v1alpha1.RemoteEnvironment) gqlsc
 	dto := gqlschema.RemoteEnvironment{
 		Name:        in.Name,
 		Description: in.Spec.Description,
-		Source: gqlschema.RemoteEnvironmentSource{
```
Comment (aszecowka) [inline components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go]: was it done on purpose, removing resource filed from graphql (and not including sourceId)

Comment (jesusreal) [reply components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go]: In adittion, shouldn't we also add `Labels JSON`?

File: components/ui-api-layer/internal/domain/remoteenvironment/re.go [hunk 2]
```diff
@@ -47,13 +47,13 @@ func New(restConfig *rest.Config, reCfg Config, informerResyncPeriod time.Durati
 	}
 
 	informerFactory := externalversions.NewSharedInformerFactory(client, informerResyncPeriod)
-	remoteEnvironmentGroup := informerFactory.Remoteenvironment().V1alpha1()
+	remoteEnvironmentGroup := informerFactory.Applicationconnector().V1alpha1()
```
[the remark this answers is not in the document]
Comment (mszostok) [reply components/ui-api-layer/internal/domain/remoteenvironment/re.go]: right, fixed

File: components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go (hunk 1 above)
Comment (mszostok) [reply components/ui-api-layer/internal/domain/remoteenvironment/re_converter.go]: It's done on purpose. Now only EventActivation has SourceId filed

**Comment by mszostok** [comment]
@jesusreal but the question is who will use that? for now, we are not mapping all fields from RE to gql model. Only those which are used on UI. Labels will be used in case of the ServiceCatalog view, so labels should be available on ServiceClasses and this will be done by Wookiee team.

**Pull request merged**